src/enchant.c

```diff
@@ -250,6 +275,21 @@ enchant_get_module_dirs (void)
 			module_dirs = enchant_slist_append_unique_path (module_dirs, module_dir);
 		}
 
+	/* Use ENCHANT_MODULE_PATH env var */
+	{
+		const gchar* env = g_getenv("ENCHANT_MODULE_PATH");
+		if (env) {
+			const gchar * path = g_filename_to_utf8(env, -1, NULL, NULL, NULL);
+			if (path) {
+				GSList *dir;
+				for (dir = _enchant_get_dirs_from_string (path); dir; dir = dir->next)
+					{
+						module_dirs = enchant_slist_append_unique_path (module_dirs, dir->data);
```

Comment:
can we get a consistent indentation style? You switch style within this chunk....

I know you did cut & paste and it was bad. that's the time to fix it.